gradlePlugin/crypto-dependency/src/main/kotlin/care/data4life/gradle/crypto/dependency/Dependency.kt

```diff
@@ -57,13 +57,13 @@ object Dependency {
             val result = Result
 
             object Util {
-                const val common = "care.data4life.hc-util-sdk-kmp:util:${Version.multiplatform.d4l.utilSdk}"
-                const val android = "care.data4life.hc-util-sdk-kmp:util-android:${Version.multiplatform.d4l.utilSdk}"
-                const val jvm = "care.data4life.hc-util-sdk-kmp:util-jvm:${Version.multiplatform.d4l.utilSdk}"
+                const val common = "care.data4life.hc-util-sdk-kmp:util:${Version.Multiplatform.D4L.utilSdk}"
```

Comment:
Why `Version.Multiplatform.D4L.utilSdk` instead of `Version.multiplatform.d4l.utilSdk`?

Reply:
I think the IDE did this while moving the package into another scope...I roll it back